docs/src/main/asciidoc/secret-management.adoc

```diff
@@ -2,9 +2,14 @@
 
 == Secret Management
 
-spring-cloud-azure-starter-keyvault-secrets adds Azure Key Vault as one of the Spring PropertySource, so secrets stored in Azure Key Vault could be easily used and conveniently accessed like other externalized configuration property, e.g. properties in files.
+About link:https://docs.microsoft.com/azure/key-vault/secrets/about-secrets[Azure Key Vault secrets], 2 features are provided:
 
-=== Dependency Setup
+ - Construct link:https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#beans-property-source-abstraction[PropertySource] which holds secrets stored in Azure Key Vault secret.
+ - Autoconfigure beans for link:https://docs.microsoft.com/azure/key-vault/secrets/quick-create-java[Azure Key Vault Secret Clients]. This feature is new added for SpringCloudAzure:4.0.
```

Comment:
This part is about secret management, so I don't think we should mention the `Autoconfigure beans` here.

Reply:
I think `secret management` contains `autoconfigure beans`. 

Customer can get secrets by PropertySource, customer can also get secrets by SecretClient. Both of them belong to `secret management`.

Reply:
The ref doc is categorized by our logical project, spring boot, spring security, spring integration, spring cloud stream, that's why I think all the autoconfigure of clients/spring data/spring integration should be in the autoconfigure part. And the ideal way to display the configuration properties are like this https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html.

Reply:
It's the same as the autoconfigure of EH client won't be introduced in the messaging part, the EH is a messaging service, but we organize the ref doc according to Spring Cloud Azure logic projects. The secret management is for something like `Spring Cloud Vault`, not for the autoconfiguration.